### ISSUE TITLE
minor installer improvements

### ISSUE THREAD

**Issue by pameyer** [IQSS/dataverse#5294]
Noticed 2 minor places to improve installer when doing a manual install:
- on some systems, `restart-domain` is less robust than `stop-domain`; `start-domain`
- if the domain isn't running, the installer attempts to sudo to configured glassfish user (which may not be in sudoers) without checking if it's running as the glassfish user

**Comment by kcondon** [comment]
@pameyer Nice avatar

**Comment by pameyer** [comment]
@kcondon thanks - I was getting confused with having >4 automaticly generated ones in different systems and decided to standardize.

**Comment by kcondon** [comment]
@pameyer Built and ran the installer. Encountered a bunch of errors when it tried to run:

./install
"my" variable $mail_server_status masks earlier declaration in same scope at ./install line 1154.
syntax error at ./install line 966, near "else"
syntax error at ./install line 970, near "}"
Global symbol "$success" requires explicit package name at ./install line 984.
Global symbol "$success" requires explicit package name at ./install line 988.
Global symbol "$success" requires explicit package name at ./install line 1071.
syntax error at ./install line 1072, near "}"
syntax error at ./install line 1119, near "}"
Can't use global @_ in "my" at ./install line 1122, near "shift @_"
syntax error at ./install line 1149, near "}"
syntax error at ./install line 1184, near "}"
./install has too many errors.

**Comment by pameyer** [comment]
@kcondon Sorry about that; I messed up my back port.  Should be fixed, I'm rechecking this branch to confirm.

**Comment by kcondon** [comment]
@pameyer OK, that works now, thanks!
Btw, I noticed some weirdness around prompts and the final DataCite message, indicating you needed to contact DataCite to get an account, even though it prompted me for the account earlier on, which I provided values for.

The weird prompts look to be kind of duplicate string combining around datacite questions:
Datacite usernameDataCite or EZID username. Only necessary for publishing / minting DOIs.[dataciteuser]
Datacite passwordDataCite or EZID account password.[datacitepassword] 
Datacite URLDataCite or EZID URL. Probably https://mds.datacite.org[https://mds.test.datacite.org]

Any thoughts on those or should they be a separate ticket?
